access/templates/access/graded_form.html

```diff
@@ -27,6 +27,14 @@
 
 			{% if result.accepted and not field.field.group_errors and field.field.max_points > 0 %}
 			<span class="badge badge-{% if field.field.name in result.error_fields %}danger{% else %}success{% endif %}">{{ field.field.grade_points }} / {{ field.field.max_points }}</span>
+			{% else %}
+			<span class="badge">
+				{% blocktrans count points=field.field.points %}
```

Comment:
Is there any reason to use `field.field.points` instead of `field.field.max_points`?

Reply:
The `field.max_points` is defined when grading the field, so it doesn't exist before the first submission.